torch/csrc/distributed/c10d/init.cpp

```diff
@@ -617,11 +625,14 @@ pair, :meth:`~torch.distributed.store.get` to retrieve a key-value pair, etc.
 
 Example::
     >>> import torch.distributed as dist
-    >>> server_store = dist.TCPStore("127.0.0.1", 0, True, timedelta(seconds=30))
-    >>> client_store = dist.TCPStore("127.0.0.1", 0, False)
+    >>> from datetime import timedelta
+    >>> # Run on process 1 (server)
+    >>> store = dist.TCPStore("127.0.0.1", 1234, 2, True, timedelta(seconds=30))
```

Comment:
Let's keep the `server_store` and `client_store` naming since there's a difference in their function and initialization